src/App.js

```diff
@@ -0,0 +1,102 @@
+import _ from "lodash";
```

Comment:
in prisma we write it like : 'import _get from loadsh/get' 
I am not sure its the right way but fyi

Reply:
Ok :)